spec/output/generator_spec_go/block_comments.x/MyXDR_generated.go

```diff
@@ -14,49 +14,65 @@ import (
   "errors"
   "io"
   "fmt"
+  "unsafe"
 
   "github.com/stellar/go-xdr/xdr3"
 )
 
+// Needed since unsafe is not used in all cases
+var _ = unsafe.Sizeof(0)
 // XdrFilesSHA256 is the SHA256 hashes of source files.
 var XdrFilesSHA256 = map[string]string{
   "spec/fixtures/generator/block_comments.x": "e13131bc4134f38da17b9d5e9f67d2695a69ef98e3ef272833f4c18d0cc88a30",
 }
 
-var ErrMaxDecodingDepthReached = errors.New("maximum decoding depth reached")
-
-type xdrType interface {
-  xdrType()
-}
-
-type decoderFrom interface {
-  DecodeFrom(d *xdr.Decoder, maxDepth uint) (int, error)
-}
-
-// Unmarshal reads an xdr element from `r` into `v`.
-func Unmarshal(r io.Reader, v interface{}) (int, error) {
-  if decodable, ok := v.(decoderFrom); ok {
-    d := xdr.NewDecoder(r)
-    return decodable.DecodeFrom(d, xdr.DecodeDefaultMaxDepth)
-  }
-  // delegate to xdr package's Unmarshal
-	return xdr.Unmarshal(r, v)
-}
-
-// Marshal writes an xdr element `v` into `w`.
-func Marshal(w io.Writer, v interface{}) (int, error) {
-  if _, ok := v.(xdrType); ok {
-    if bm, ok := v.(encoding.BinaryMarshaler); ok {
-      b, err := bm.MarshalBinary()
-      if err != nil {
-        return 0, err
-      }
-      return w.Write(b)
-    }
-  }
-  // delegate to xdr package's Marshal
-  return xdr.Marshal(w, v)
+          var ErrMaxDecodingDepthReached = errors.New("maximum decoding depth reached")
+
+          type xdrType interface {
+            xdrType()
+          }
+
+          type decoderFrom interface {
+            DecodeFrom(d *xdr.Decoder, maxDepth uint, maxAllocSize int) (int, error)
+          }
+
+          // Unmarshal reads an xdr element from `r` into `v`.
+          func Unmarshal(r io.Reader, v interface{}) (int, error) {
+            return UnmarshalWithMaxAllocSize(r, v, 0)
+          }
+
+          // Unmarshal reads an xdr element from `r` into `v`.
```

Comment:
in the parse/generator, can the comment be updated to state behavior when `maxAllocSize` parameter is `0`, does that equate to unbounded then?